cmd/kubelet/app/options/options.go

```diff
@@ -416,6 +416,7 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.BoolVar(&f.KeepTerminatedPodVolumes, "keep-terminated-pod-volumes", f.KeepTerminatedPodVolumes, "Keep terminated pod volumes mounted to the node after the pod terminates.  Can be useful for debugging volume related issues.")
 	fs.MarkDeprecated("keep-terminated-pod-volumes", "will be removed in a future version")
 	fs.BoolVar(&f.EnableCAdvisorJSONEndpoints, "enable-cadvisor-json-endpoints", f.EnableCAdvisorJSONEndpoints, "Enable cAdvisor json /spec and /stats/* endpoints.")
+	// TODO: Remove this flag in 1.20+.  https://github.com/kubernetes/kubernetes/issues/68522
```

Comment:
also add to pkg/kubelet/cadvisor/cadvisor_linux.go, line 35-36 (replacing the link to the stale closed issue that's already there)?

Reply:
done